### PR TITLE
Update troubleshooting.md

### PR DESCRIPTION
**Pull request by lucasdigital**
Includes bonus "delayBetweenScanRounds" parameter change which was uncovered during discussion with engineer.

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -155,6 +155,37 @@ The notification message is sent 10 minutes after contact with the Kafka service
 3. If there are very large numbers of queued changes we recommend that you reset the migration. It'll be faster and more reliable to rescan the source than attempt to continue the stalled migration.
 
 
+### Hive Migrator connection to the source or target timed-out
+
+Hive metadata migrations are set to fail if connections to either the target or source agent are lost for more than 20 minutes. This fix for these failures is to restart the affected migrations.
```
Comment (stepgerner) [inline docs/troubleshooting.md:160]: Should this read "the fix for these failures"? Maybe for that matter, "To fix one of these failures, restart the affected migrations".

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -155,6 +155,37 @@ The notification message is sent 10 minutes after contact with the Kafka service
...
+
+If migrations continue to fail due to this timeout, consider increasing the `connectionRetryTimeout` parameter.
```
Comment (stepgerner) [inline docs/troubleshooting.md:162]: Tiniest nitpick, maybe add a colon onto the end of this sentence to illustrate to an impatient reader that we're about to tell them how to do just that.

**Review by stepgerner — COMMENTED** [review]
Two comments!

File: docs/troubleshooting.md (hunk 1 above)
Comment (lucasdigital) [reply docs/troubleshooting.md:160]: Ah yeah, lets cut it back further with "Fix any failures by restarting the affected migrations"

File: docs/troubleshooting.md (hunk 2 above)
Comment (lucasdigital) [reply docs/troubleshooting.md:162]: That's fair, I think mostly from a consistency point-of-view. Now updated.

**Pull request merged**